.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -98,6 +110,7 @@ body:
         - "Dask"
         - "dashboards"
         - "Grafana"
+        - "Argo"
```

Comment:
There is a typo on line 104 (just above here): `Is this issue related to onay of the Nebari integrations?`